src/Framework/ConfigResolverAwareTrait.php

```diff
@@ -11,6 +11,14 @@ trait ConfigResolverAwareTrait
 {
     private ?AbstractConfig $config = null;
 
+    /**
+     * Syntax sugar to access the config from static methods.
+     */
+    public static function config(): AbstractConfig
+    {
+        return (new static())->getConfig();
+    }
+
     public function getConfig(): AbstractConfig
```

Comment:
I am not sure about naming the method `config()` when it is static and `getConfig()` when it is not.
Maybe it would be better to unify the name in both situations?

Reply:
The thing is that you cannot have the same func name for both (non-static and static function). Unless we use magic methods like __call and __staticCall ... 🤔 

> I will investigate this part. Thanks for bringing this up 🙇🏼